### ISSUE TITLE
Setting cursor when clicking on content area which does not have content

### ISSUE THREAD

**Issue by archhhh** [accordproject/template-studio-v2#126]
**Is your feature request related to a problem? Please describe.**

Currently when you try to click on the content area which doesn't have any content, cursor disappears, or if you are out of focus and again try to click on the area, cursor does not appear.

**Describe the solution you'd like**

Setting cursor after the last character available, like it is in editors like Visual Studio Code or Microsoft Word.

**Comment by archhhh** [comment]
let me know, if the description is not clear. i am not a native english speaker and to be honest i am having hard time explaining this feature in a clear way.

**Comment by irmerk** [comment]
Yes I think I understand. So in the [Netlify demo](https://accordproject-studio.netlify.com/), clicking inside or immediately after `Welcome to Template Studio! Edit this text to get started.` works, but clicking anywhere else in the white area of the editor results in nothing happening. Correct?

**Comment by archhhh** [comment]
Yes, that's what I meant

**Comment by irmerk** [comment]
Moving this issue to `markdown-editor`, because I believe it stems from there (the same behavior happens in the [Netlify for it](https://accordproject-markdown-editor.netlify.com/examples/))
